package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/whitphx/vscode-emacs-mcx"
   },
   "engines": {
-    "vscode": "^1.63.0"
+    "vscode": "^1.89.0"
```

Comment:
Note: `workbench.action.quickTextSearch` was introduced in 1.89: https://code.visualstudio.com/updates/v1_89#_quick-search